io.go

```diff
@@ -12,6 +12,9 @@ import (
 
 // copy recursively copies src into dst with src's file modes.
 func copy(src, dst string) error {
+	if !filepath.IsAbs(src) {
```

Comment:
This is actually done for you in `filepath.Abs()`, so no need :+1:

Reply:
Done